agents/src/ipc/job_main.ts

```diff
@@ -110,10 +110,10 @@ if (process.send) {
   process.on('SIGINT', () => {});
 
   await once(process, 'message').then(([msg]: IPCMessage[]) => {
-    if (msg.case !== 'initializeRequest') {
+    if (msg!.case !== 'initializeRequest') {
       throw new Error('first message must be InitializeRequest');
     }
-    initializeLogger(msg.value.loggerOptions);
+    initializeLogger(msg!.value.loggerOptions);
```

Comment:
the double `!` here and two lines above could be reduced to a single check of msg